scripts/drtprod

```diff
@@ -311,8 +311,7 @@ EOF"
           --gce-use-spot \
           --nodes 15 \
           --gce-machine-type n2-standard-16 \
-          --local-ssd=true \
-          --gce-local-ssd-count 16 \
+          --pd=6000 \
```

Comment:
I don't see the `--pd` flag in the help for `roachprod create`.  The way I would have thought we'd have done this is by using:

```
--local-ssd=false
--gce-pd-volume-size=6000
--gce-pd-volume-type=pd-standard
```